pkg/artifactory/resource_artifactory_single_replication_config_test.go

```diff
@@ -34,7 +34,7 @@ func mkTclForRepConfg(name, cron, url, proxy string) string {
 		name,
 		cron,
 		url,
-		os.Getenv("ARTIFACTORY_USERNAME"),
+		rtDefaultUser,
```

Comment:
This works but I prefer more verbose name, like `rtDefaultUsername` or even `artifactoryTestUsername`

Reply:
okay, `rtDefaultUsername` then